sdk/digitaltwins/Azure.DigitalTwins.Core/src/Customized/Models/BulkImportJob.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.DigitalTwins.Core
+{
+    [CodeGenModel("BulkImportJob")]
+    public partial class BulkImportJob
```

Comment:
Do we want to rename this and all other files (e.g BulkImportJobCollection, Serialization, etc )to not include "Bulk" ? Could change this to just be "ImportJob"

cc @abhinav-ghai

Reply:
Yes, let's rename BulkImportJob to ImportJob in all places.